tasks/main.yml

```diff
@@ -157,6 +165,80 @@
     mode=0755
   become_user: "{{ project_user }}"
 
+- name: configure celery and celery beat
+  template: src=celery.conf
+            dest=/etc/supervisor/conf.d/{{ project_name }}-celery-{{ item.name }}.conf
+            owner=root
+            group=root
+            mode=0600
+  with_items:
+    - { name: "default", command: "worker", flags: "--loglevel=INFO" }
+    - { name: "beat", command: "beat", flags: "--schedule={{ root_dir }}/celerybeat-schedule.db --pidfile={{ root_dir }}/celerybeat.pid --loglevel=INFO" }
+  notify:
+    - restart celery
+    - restart celery-beat
+
+- name: ensure celery is running
+  supervisorctl: name={{ project_name }}-celery-default state=started
+
+- name: ensure celery-beat is running
+  supervisorctl: name={{ project_name }}-celery-beat state=started
+
+- name: add nodejs repo
+  apt_repository: repo="deb https://deb.nodesource.com/node_4.x {{ ansible_distribution_release }} main"
+                  state=present
+                  filename='nodesource'
+
+- name: add nodejs repo key
+  apt_key: data="{{ lookup('file', 'nodesource.pub') }}" state=present
+
+- name: ensure that nodejs-legacy is not installed
+  apt: pkg=nodejs-legacy state=absent
+
+- name: install nodejs
+  apt: pkg=nodejs state=latest
+
+- name: npm install packages
+  npm: path={{ source_dir }} state=latest
+  become_user: "{{ project_user }}"
+
+- name: npm run build
+  command: npm run build
+  args:
+    chdir: "{{ source_dir }}"
+  become_user: "{{ project_user }}"
+
+# TODO: find out if the latest gunicorn is ok, or if we really do want
+# to pin to a version in the 19.1 series.
+- name: install gunicorn
+  pip: name=gunicorn
+       state=latest
+       virtualenv={{ venv_dir }}
+       virtualenv_python=/usr/bin/python{{ python_version }}
+  become_user: "{{ project_user }}"
+
+- name: configure gunicorn
+  template: src=gunicorn.conf
+            dest=/etc/supervisor/conf.d/{{ project_name }}-gunicorn.conf
+            owner=root
+            group=root
+            mode=0600
+  notify:
+    - reload supervisor
+
+- name: ensure gunicorn is running
+  supervisorctl: name={{ project_name }}-server state=started
+
+# TODO: let connections to 8000 through the firewall if we are load-balancing.
+
+- name: install less
+  npm: name=less version={{ less_version }} state=present
+
+- name: symlink less
+  file: src=/usr/bin/lessc
+        dest=/usr/local/bin/lessc
+        state=link
```

Comment:
This symlink surprises me. If we have `lessc` in `/usr/bin`, why would we need a symlink in `/usr/local/bin`?